assemblies/assembly-dashbuilder.html

```diff
@@ -0,0 +1,501 @@
+<!DOCTYPE html>
```

Comment:
Is adding `assembly-dashbuilder.html` to the `assemblies` folder intentional? I may be mistaken but is this perhaps a fragment from a doc build that wasn't git-ignored?

Reply:
No, just forgot to delete. Thanks @csherrar!